connectors/ccmixter.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+setupConnector();
+
+function setupConnector() {
+	if (isBetaPlayer()) {
+		setupBetaPlayer();
+	} else if (isTuneTrackPlayer()) {
+		setupTuneTrackPlayer();
+	} else {
+		setupH5Player();
```

Comment:
Should we add a check for the H5 page so we can console log if we fail to match or is this generic enough to handle all cases?